opencog/reasoning/RuleEngine/rules/relex-to-logic/TOBE.scm

```diff
@@ -0,0 +1,73 @@
+(define TOBE
+	(BindLink
+		(ListLink
+			(TypedVariableLink
+				(VariableNode "$a-parse")
+				(TypeNode "ParseNode")
+			)
```

Comment:
On Thu, Apr 30, 2015 at 2:34 PM, anitzkin <notifications@github.com> wrote:

> In opencog/reasoning/RuleEngine/rules/relex-to-logic/TOBE.scm
> <https://github.com/opencog/opencog/pull/1575#discussion_r29406442>:
>
> > @@ -0,0 +1,73 @@
> > +(define TOBE
> > +	(BindLink
> > +		(ListLink
> > +			(TypedVariableLink
> > +				(VariableNode "$a-parse")
> > +				(TypeNode "ParseNode")
> > +			)
>
> Gosh. Okay. Agaiin, i was just copying your examples; i figure that was
> what they were for; to show us what you wanted them to look like!
>


... the world is much subtler than that, young apprentice !!!!